docs/sphinx/source/installation.rst

```diff
@@ -34,49 +33,153 @@ There are many ways to install Python on your system, but the Anaconda
 Scientific Python distribution provides by far the easiest way for new
```

Comment:
There are also automated Ubuntu builds.
But since many of teh ubuntu packages are quite outdated (e.g. jupyter) this may not be relevent. Once situation changes at their end, we can update the text here.